pom.xml

```diff
@@ -72,6 +72,12 @@
 			<version>${testcontainers.version}</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>net.java.dev.jna</groupId>
+			<artifactId>jna</artifactId>
+			<version>5.12.0</version>
+			<scope>test</scope>
+		</dependency>
```

Comment:
you do not have a mac with a m1 processor ;)
Testcontainer does not work otherwise: https://github.com/testcontainers/testcontainers-java/issues/3834

Reply:
Ah, what a shit. I added a comment. 
OK